drivers/hmis/app/graphql/types/hmis_schema/has_health_and_dvs.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  module HmisSchema
+    module HasHealthAndDvs
+      extend ActiveSupport::Concern
+
+      class_methods do
+        def health_and_dvs_field(name = :events, description = nil, **override_options, &block)
+          default_field_options = { type: HmisSchema::HealthAndDv.page_type, null: false, description: description }
+          field_options = default_field_options.merge(override_options)
+          field(name, **field_options) do
+            instance_eval(&block) if block_given?
+          end
+        end
+      end
+
+      def resolve_health_and_dvs(scope = object.health_and_dvs, **args)
+        apply_arguments(scope, **args)
+      end
+
+      private
+
+      def apply_arguments(scope)
```

Comment:
```suggestion
      def scoped_health_and_dvs(scope)
```

Reply:
thanks, regex find and replace missed these